Mathlib/RingTheory/Etale/Kaehler.lean

```diff
@@ -333,7 +333,7 @@ lemma tensorH1CotangentOfIsLocalization_toLinearMap
     LinearEquiv.coe_coe, LinearMap.liftBaseChange_tmul, one_smul]
   simp only [tensorH1CotangentOfIsLocalization, Generators.toExtension_Ring,
     Generators.toExtension_commRing, Generators.self_vars, Generators.toExtension_algebra₂,
-    Generators.self_algebra, AlgHom.toRingHom_eq_coe, Extension.tensorH1Cotangent,
```

Comment:
In theory this should be deprecated, but it's almost certainly not worth the effort.

Reply:
Should I just remove `Generators.self_algebra`?